### ISSUE TITLE
Start Script

### ISSUE THREAD

**Issue by lhncad** [stummyhurt/twitchstreamwatcher#1]
I tried running both start scripts and they both close right after opening. Is there anything I can do to fix this?

**Comment by seansusmilch** [comment]
I've noticed that with less and less streamers streaming valorant, this script might run into array index out of bounds errors.

If you want to prevent start.bat from closing, you can edit, and add `PAUSE` at the bottom of it.

**Comment by lhncad** [comment]
Is there anyway to stop the out of bounds error cause I get it almost
every hour?

**Comment by seansusmilch** [comment]
The quick fix for that would be to edit `watchStream.py` and find this line near the bottom
```python
elements[random.randint(0,100)].click()
```
and change it to
```python
elements[random.randint(0,len(elements))].click()
```

**Comment by seansusmilch** [comment]
I made a new release that hopefully solves this issue and others that I've found
https://github.com/stummyhurt/twitchstreamwatcher/releases/tag/v0.2

**Comment by lhncad** [comment]
Ok, thanks for the update.

**Comment by lhncad** [comment]
I modified the script to only watch one streamer. Will this have negative
impact on getting a key?

**Comment by seansusmilch** [comment]
You're probably good. I see people talking about a 2 hour threshold for being eligible for a key but I got a drop with this script when it was watching streamers for 2-36 minutes. When I first made this script, I never heard anything about a 2 hour threshold.

Several of my friends got their keys by watching a stream overnight on their phone or laptop, so I think watching 1 streamer for a long time is a much simpler approach that can be just as effective.
